content/handbook/product/product-processes/customer-issues-prioritization-framework.md

```
@@ -0,0 +1,196 @@
+stale: false
+---
+
+## なぜこのフレームワークが存在するのか
+
+GitLab のプロダクトチームは、サポートチケット、セールス会話、CSM とのやり取り、直接の顧客 Issue など、複数のチャンネルを通じて顧客フィードバックを受け取ります。一貫性のある優先順位付けシステムがないと、重要な顧客ニーズがノイズの中で失われ、チームは顧客の成功にとって何が最も重要かについて足並みを揃えるのに苦労します。
+
+このフレームワークは、GitLab チームメンバーが顧客への影響をプロダクトチームに伝えるための共通言語を提供し、セールスを阻むもの、更新を危うくするもの、約束された成果を表す Issue が適切な注意を受けられるようにします。
+
+## フィードバックを提供する場所
+
+Issue に関する顧客フィードバックを提供する推奨方法は、[以下に記載](#quick-start) されているプロセスに従って **内部コメント** を使用することです。内部コメントは顧客のプライバシーを保護し、機密のビジネス情報が公に表示されることを防ぎます。
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)